src/deprecation.h

```diff
@@ -10,10 +10,19 @@
 // * A warning is shown during the 2 weeks' worth of blocks prior to shut down.
 static const int APPROX_RELEASE_HEIGHT = 625500;
 static const int WEEKS_UNTIL_DEPRECATION = 16;
-static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 24);
+// static const int DEPRECATION_HEIGHT = APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 24);
+
+// TEMPORARY!
+// This is hardcoded to a height that is roughly 10 weeks following the
+// activation of Blossom on mainnet, which is itself almost 6 weeks following
+// this release. This is done because the calculation of the deprecation height
+// does not account for the change in block target spacing; after Blossom has
+// activated, this should be removed and the previous line fixed to use
+// APPROX_RELEASE_HEIGHT + (WEEKS_UNTIL_DEPRECATION * 7 * 24 * 48)
+static const int DEPRECATION_HEIGHT = 734240;
 
 // Number of blocks before deprecation to warn users
-static const int DEPRECATION_WARN_LIMIT = 14 * 24 * 24; // 2 weeks
+static const int DEPRECATION_WARN_LIMIT = 14 * 24 * 48; // 4 weeks
```

Comment:
No this is still two weeks. (14 days of 24 hours with 48 expected blocks per hour, since those two weeks are entirely after Blossom activation.)

Reply:
Doh! Thanks.